backend/.env

```diff
@@ -5,10 +5,14 @@ VANTAA_AIKUISOPISTO_PATH=http://www.sivistysvantaa.fi/vantaanaikuisopisto/artikk
 VANTAA_NUORTEN_PATH=http://www.sivistysvantaa.fi/nuortenvantaa/artikkelit/jutut/uutiset/rss.xml
 VANTAA_KAUPUNGINMUSEO_PATH=http://www.sivistysvantaa.fi/vantaankaupunginmuseo/artikkelit/Info/uutiset/rss.xml
 VANTAA_EVENTS_PATH=https://backend.eventz.today/feeds/channel/vantaa/rss-strict?lang=fi&count=30&days=7
```

Comment:
This should just be Base path. No query params. Inject params in your caller code

Reply:
yes will do this one.